spec/index.md

```diff
@@ -445,10 +445,21 @@ possible (and thus not required), since a client with no secret cannot prove its
 identity when it issues a call. (The end-to-end system can still be secure
 because the client comes from a known, https protected endpoint specified and
 enforced by the redirect uri.)  For <span class="label
-label-primary">confidential apps</span>, an `Authorization` header using HTTP
-Basic authentication is required, where the username is the app's `client_id`
-and the password is the app's `client_secret` (see
-[example](basic-auth-example/index.html)).
+label-primary">confidential apps</span>, authentication is required.
```

Comment:
```suggestion
label-primary">confidential apps</span>, authentication is required. Clients SHOULD use JWT assertions and MAY use Client Password authentication.
```

Reply:
This is "outdated" now so I can't auto-apply, but I'm manually incorporating your suggestion @isaacvetter